spec/support/feature_helpers.rb

```diff
@@ -353,18 +356,22 @@ def form_is_filled_in_by_form_filler(live_form_link, skip_question: false, confi
     logger.info "When a form filler has submitted their answers"
     logger.info "Then I can see their submission in my email inbox"
 
-    form_submission_email = wait_for_notification(submission_email_reference)
-
-    logger.info "And I can see their answers"
-    if skip_question
-      expect(form_submission_email.body).to have_content selection_question
-      expect(form_submission_email.body).to have_content "Yes"
+    if ENV.fetch('SES_SUBMISSIONS', true)
+      check_file_upload_submission
```

Comment:
Now that we're using this check for all submissions we should rename this helper method to maybe `check_submission`?